### PR TITLE
Alec/tfac 357

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
@kmjennison I think it's just this?

File: src/pages/500.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import Error from 'next/error'
+
+export default function Custom500() {
+  return <Error statusCode={500} />
```
Comment (amaliwanag2) [inline src/pages/500.js:5]: @kmjennison saw this ticket was a 1 so I copied the return from the 404 page.  Is this what you were thinking for the 500 page?

Comment (kmjennison) [reply src/pages/500.js:5]: Yep! Just want a branded, less jarring error view.

**Comment by amaliwanag2** [comment]
@kmjennison verified custom error page shows on 500

**Review by kmjennison — APPROVED** [review]
LGTM! 🍝

**Pull request merged**